packages/react/src/components/Modal/Modal.mdx

```diff
@@ -1,15 +1,325 @@
-import { Props } from '@storybook/addon-docs/blocks';
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
```

Comment:
Can we just rearrange this to be consistent with the other docs? 

![Screen Shot 2020-09-25 at 10 27 54 AM](https://user-images.githubusercontent.com/11928039/94297736-d9b4ee00-ff19-11ea-9ee2-786c486f3522.png)